charts/argo-rollouts/templates/dashboard/deployment.yaml

```diff
@@ -45,6 +45,7 @@ spec:
         {{- toYaml . | nindent 8 }}
       {{- end }}
       serviceAccountName: {{ include "argo-rollouts.serviceAccountName" . }}-dashboard
+      automountServiceAccountToken: {{ .Values.dashboard.serviceAccount.automount }}
```

Comment:
Since we are setting the field on the `kind: Deployment` rather than on the `kind: ServiceAccount` can we use `{{ .Values.dashboard.automountServiceAccountToken }}` instead?